src/announcement/AnnouncementController.ts

```diff
@@ -23,43 +23,43 @@ interface NotificationMap {
 /**
  * A map of notification ids to StateNotification objects
  */
-export interface StateNotificationMap {
+export interface StateAnnouncementMap {
   [id: number]: StateNotification;
```

Comment:
Nit: For consistency it would be great to update these as well. Any remnant use of the term "notification" could get confused with the new notification system.